src/foam/nanos/auth/LastModifiedAware.js

```diff
@@ -13,7 +13,10 @@ foam.INTERFACE({
       class: 'DateTime',
       name: 'lastModified',
       documentation: 'Last modified date',
-      storageOptional: true
+      storageOptional: true,
+      javaCompare: 'return 0;',
```

Comment:
Will this still allow us to sort/orderBy lastModified?

Reply:
reverted in final version.